docs/package.json

```diff
@@ -5,7 +5,8 @@
     "dependencies": {
         "@reach/component-component": "^0.1.3",
         "@retailmenot/anchor": "^0.0.1-beta.8",
-        "@xstyled/styled-components": "^1.5.2",
```

Comment:
We're evergreen, so we can't stamp versions.

Reply:
Pulled in latest from master with this fix.